### PR TITLE
introduce appendLines method in IO.

### PR DESCRIPTION
**Pull request by trilleplay**
Adds appendLines to the IO api, which lets users easily append a sequence of strings to a file that already exists.

Solves #45 .

### CODE REVIEW

**Comment by bjornregnell** [comment]
I guess we should have trailing "\n" in the line-writers.

**Comment by trilleplay** [comment]
Added a trailing newline @bjornregnell

File: src/main/scala/introprog/IO.scala [hunk 1]
```diff
@@ -51,7 +51,7 @@ object IO:
     * @param enc the encoding of the file.
     * */
   def saveLines(lines: Seq[String], fileName: String, enc: String = "UTF-8"): Unit =
-    saveString(lines.mkString("\n"), fileName, enc)
+    saveString(lines.mkString("\n") + "\n", fileName, enc)
```
Comment (bjornregnell) [inline src/main/scala/introprog/IO.scala:54]: I think if lines are empty it should not write so better use `mkString("", "\n", "\n")` see docs for mkString to double-check the behavior or test in repl.

File: src/main/scala/introprog/IO.scala [hunk 2]
```diff
@@ -47,7 +51,34 @@ object IO:
...
+
+  /**
+    * Appends `lines` to the text file `fileName` using encoding `enc`.
+    *
+    * @param lines the lines to append to the file.
+    * @param fileName the path of the file.
+    * @param enc the encoding of the file.
+    * */
+  def appendLines(lines: Seq[String], fileName: String, enc: String = "UTF-8"): Unit =
+    appendString(lines.mkString("\n"), fileName, enc)
```
Comment (bjornregnell) [inline src/main/scala/introprog/IO.scala:81]: also here it should be `mkString("","\n","\n")` I think, but check that it works as expected.

Comment (bjornregnell) [reply src/main/scala/introprog/IO.scala:81]: I'll merge this and then fix the correct behavior wen empty in a later commit.

**Pull request merged**